src/MyHealth/index.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { FunctionComponent, useState } from "react"
+import { View, StyleSheet, ScrollView } from "react-native"
+import SegmentedControl from "@react-native-community/segmented-control"
+import { useTranslation } from "react-i18next"
+
+import { useStatusBarEffect } from "../navigation"
+import { GlobalText, StatusBar } from "../components"
+import Today from "./Today"
+import OverTime from "./OverTime"
+
+import { Typography, Colors, Spacing } from "../styles"
+
+const MyHealthScreen: FunctionComponent = () => {
+  useStatusBarEffect("dark-content", Colors.primaryLightBackground)
+  const { t } = useTranslation()
+  const [selectedIndex, setSelectedIndex] = useState<number>(0)
```

Comment:
This is really minor, but what do you think about naming this in terms of the UI, rather than the underlying implementation `SegmentedControl`? Perhaps with an enum, which is a `number` under the hood? Could be something like
```
enum MyHealthScreen {
   Today,
    Overtime
}

const [activeTab, setActiveTab] = useState<MyHealthScreen>(MyHealthScreen.Today)

const determineTodayStyles = () => {
   return activeTab === MyHealthScreen.Today ? {} : style.hidden
}
```

Reply:
Happy to do this, maybe in a second pass since we do `need` an actual index for the segmented control, and even if the types agree with an enum, it feels like a stretch to derive anevent argument into an enum. Thoughts?